### PR TITLE
Adds info for kick off to top banner

### PR DESCRIPTION
**Pull request by liamshatzel**
# Description

Added info for kick-off to top banner with link to vikelabs.ca. Would it be good to also change banner color on kick off info to something like red??

## Checklist

- [x] The code follows all style guidelines.
- [x] The code passes all required tests.
- [ ] The code is documented.
- [ ] The code includes tests.
- [x] I have self-reviewed my changes and have done QA.

### CODE REVIEW

**Review by JoshoTheMosho — APPROVED** [review]
LGTM!

File: src/common/header/containers/HeaderContainer.tsx [hunk 1]
```diff
@@ -63,8 +63,24 @@ export interface HeaderProps {
 export function HeaderContainer({ onSearchChange }: HeaderProps): JSX.Element {
   const smallScreen = useSmallScreen();
   const tips: Array<JSX.Element> = smallScreen
-    ? [<Text>⚠️ Mobile is currently in beta.</Text>]
+    ? [
+        <Text>⚠️ Mobile is currently in beta.</Text>,
+        <Text>
+          ⚠️ Want to build something like CourseUp? Come to the VikeLabs{' '}
+          <a href="https://vikelabs.ca/kickoff" style={{ textDecoration: 'underline' }} target="_blank">
```
Comment (JoshoTheMosho) [inline src/common/header/containers/HeaderContainer.tsx:70]:
```suggestion
          <a href="https://vikelabs.ca/kickoff" style={{ textDecoration: 'underline' }} target="_blank" rel="noreferrer">
```

File: src/common/header/containers/HeaderContainer.tsx [hunk 2]
```diff
@@ -63,8 +63,24 @@ export interface HeaderProps {
...
+            Kick-Off
+          </a>{' '}
+          on September 18th! 🚀
+        </Text>,
+      ]
     : [
+        <Text>
+          ⚠️ Want to build something like CourseUp? Come to the VikeLabs{' '}
+          <a href="https://vikelabs.ca/kickoff" style={{ textDecoration: 'underline' }} target="_blank">
```
Comment (JoshoTheMosho) [inline src/common/header/containers/HeaderContainer.tsx:79]:
```suggestion
          <a href="https://vikelabs.ca/kickoff" style={{ textDecoration: 'underline' }} target="_blank" rel="noreferrer">
```

**Review by JoshoTheMosho — CHANGES_REQUESTED** [review]
Just noticed failing tests for security issue

**Review by szeckirjr — APPROVED** [review]
besides what Josh said, LGTM too

File: src/common/header/containers/HeaderContainer.tsx (hunk 1 above)
Comment (liamshatzel) [reply src/common/header/containers/HeaderContainer.tsx]: 71d7f603d25c7778a7bf9e0256b020222886f9b7

File: src/common/header/containers/HeaderContainer.tsx (hunk 2 above)
Comment (liamshatzel) [reply src/common/header/containers/HeaderContainer.tsx]: c772b0df2ea057a31a4202eb1fa217a40c43f6a7

**Comment by szeckirjr** [comment]
run `npm run lint:fix` to fix CI linting errors btw 👀

**Review by szeckirjr — APPROVED** [review]
kick off on sep 18th 🚀

**Review by JoshoTheMosho — APPROVED** [review]
LGTM

**Pull request merged**